internal/datastore/required_user.go

```diff
@@ -66,7 +66,7 @@ func (r *requiredUser) update(data map[string]json.RawMessage) error {
 	r.users = make(map[int][]string)
 
 	for k, v := range r.requiredUser {
-		for _, uid := range v {
+		for uid := range v {
 			// Make sure each perm is only once in the slide. For smal slides,
 			// this is faster then using a set.
```

Comment:
Please remove line 72 to line 79. This lines where necessary to remove double entries of the same uid. Since you are using a set, this is not necessary anymore.